CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## unreleased
 
+## 0.7.0 - 2022-02-09
+
 ### Changed
```

Comment:
I would maybe make it `### Fixed`? And add smt like "dispatcher not being Send", so it's clear that was the thing that we were fixing

Reply:
Done.